wire-ios-data-model/Source/MLS/Migration/ProteusToMLSMigrationCoordinator.swift

```diff
@@ -209,3 +255,9 @@ public class ProteusToMLSMigrationCoordinator: ProteusToMLSMigrationCoordinating
     }
 
 }
+
+extension ZMUser {
+    var supportedProtocols: [MessageProtocol] {
+        return [.mls, .proteus]
+    }
+}
```

Comment:
This is temporary until John's work on 1on1 conversations is merged to develop and then we can pull the changes into the migration branch.

Reply:
Done!